pages/task/read/content/confirm-hba.js

```diff
@@ -10,8 +10,17 @@ module.exports = merge({}, baseContent, {
     establishment: {
       label: 'Establishment:'
     },
+    currentEstablishment: {
+      label: 'Current establishment:'
+    },
+    proposedEstablishment: {
+      label: 'Proposed establishment:'
+    },
     applicant: {
-      label: 'Applicant:'
+      label: 'PPL holder:'
+    },
+    proposedApplicant: {
+      label: 'Proposed PPL licence holder:'
```

Comment:
@edemirbag this is the only change, and name used.

Reply:
noted - will be easier to discuss this over a call

Reply:
For reference:

- https://github.com/UKHomeOffice/asl-pages/commit/3473b29f371490f7b45aa4da51cdef5d79da873b
- https://github.com/UKHomeOffice/asl-pages/commit/4a5ad8ce89a52f063b8e47d9a72baa09a6fe0058